elogia_sale/models/elogia_sale.py

```diff
@@ -377,21 +465,25 @@ def generated_account_move(self, record, env_mov, obj_setting):
             'move_type': 'entry',
             'line_ids': []
         }
+        billed_currency = record.billed_revenue_ml
+        if record.currency_id != record.company_id.currency_id:
+            billed_currency = record.billed_revenue_ml / record.currency_id.rate
         for item in list_setting:
             vals['line_ids'].append(((0, 0, {
                 'account_id': item['value_d'].id if item['key'] == 'debit' else item['value_c'].id,
                 'name': record.name,
                 'partner_id': record.client_id.id,
-                'debit': record.billed_revenue_ml if item['key'] == 'debit' else 0,
-                'credit': record.billed_revenue_ml if item['key'] == 'credit' else 0,
+                'debit': billed_currency if item['key'] == 'debit' else 0,
```

Comment:
Así queda más bonito:
tem['key'] == 'debit'  and billed_currency or 0

Reply:
Bien